### PR TITLE
Add a last rescraped time to sources

### LINKED ISSUE #788: Add a "rescraped_at" column to sources_source table?
User Story:
When there is a volume issue for a source, my first impulse is to want to know when it was last rescraped for feeds!

An ancillary bit of data is for the feeds table: when it was last seen in a rescape (ie; when was the RSS file or sitemap last discoverable or advertised), but I'm prone to adding columns to table (look at the rss-fetcher feeds table):
```
id                  | 905793
sources_id          | 665279
name                | La Provincia
url                 | http://www.laprovinciacr.it/rss.jsp?sezione=503
active              | t
last_fetch_attempt  | 2024-09-16 10:53:40.698808
last_fetch_success  | 2024-09-16 10:53:40.698808
last_fetch_hash     | 8091191156edd9fd2906c551378c89a5
last_fetch_failures | 0
created_at          | 2022-12-23 17:44:04.736466
http_etag           | 
http_last_modified  | 
next_fetch_attempt  | 2024-09-17 10:53:41.189757
queued              | f
system_enabled      | t
update_minutes      | 
http_304            | 
system_status       | Working
last_new_stories    | 2024-05-29 03:03:12.928357
rss_title           | La Provincia
poll_minutes        | 1440
```

### PR DESCRIPTION
**Pull request by Evan-Leon**
## Add last rescraped time to a source
 - source get this time edited at the end of Source._scrape_source

### Added to source page
Screen Shot 2024-11-26 at 10 37 35 AM

### Added to source list for collection page
Screen Shot 2024-11-26 at 10 39 08 AM

- as noted in #788 comments, more can be added in future utilizing this column

closes #788

### CODE REVIEW

**Comment by Evan-Leon** [comment]
After comments on the issue going to add last_rescrape_details or something of the sort, to record the feeds added message

**Comment by Evan-Leon** [comment]
## added last_rescraped_msg column to Source table 
### added message to source show page
Screen Shot 2024-12-03 at 11 24 08 AM
Additionally made the changes Phil suggested in #788 of adding a line in for logging

File: mcweb/backend/sources/models.py [hunk 1]
```diff
@@ -310,8 +321,11 @@ def process_urls(from_: str, urls: list[str]):
             process_urls(sitemaps, gnews_urls)
 
         # after many tries to give a summary in english:
-        add_line(f"{added}/{total} added, {confirmed}/{old} confirmed")
-
+        summary = f"{added}/{total} added, {confirmed}/{old} confirmed"
+        add_line(summary)
+        logger.info("%s", summary)
```
Comment (philbudne) [inline mcweb/backend/sources/models.py:326]: Thank you for addding the log message!!!

File: mcweb/backend/sources/models.py [hunk 2]
```diff
@@ -250,7 +261,7 @@ def add_line(line):
 
         # per-source header line
         add_line(f"Scraped source {source_id} ({name}), {homepage}")
-
+
```
Comment (philbudne) [inline mcweb/backend/sources/models.py:264]: added invisible whitespace??

**Review by philbudne — APPROVED** [review]
Thanks for asking for review comments, only note was some extraneous whitespace!

**Pull request merged**